### DISCUSSION TITLE
Episode Select font

### DISCUSSION THREAD

**Discussion by BVH1999** [Owlet7/wadfusion#33] [General]
Any way to get the episode selection screen to show those cool red letters again?
This is so close to a perfect game, I'd love some help.

This is such a small problem, but it was worth making a Github account for.

**Comment by Owlet7** [comment]
It's something that's planned, see #6. There are too many episodes now to fit with the big font on one screen, so we'll group them into several new menus.

Thanks for checking out WF! I appreciate it being called close to perfect, but yeah, there are still [a few things that need to be implemented](https://github.com/Owlet7/wadfusion/blob/master/TODO.md) before I can consider it "finished".

**Reply by BVH1999** [reply]
thanks so much for answering. I think it's very cool you're working so hard.

**Reply by SpacedOutDude** [reply]
I found away to display all the episodes, If you Place this into MENUDEF It will allow all the 13 episodes to display in original big font

ListMenu "EpisodeMenu"
{
    Size 400, 270
    Font "BigFont", "RED"
    Linespacing 18  
    StaticPatch 100, 2, "M_EPISOD"
    Position 80, 30   
    ForceList
}

Just take 20 off the "y" value of Size for every episode missing, so for 12 episodes, Size 400, 250, for 11 Size 400, 230.... Its only needed for more than 10 episodes, anything under that are displayed fine. Perhaps you could add it to the build process, to add the set of numbers depending on number of Iwads detected or something like that. Here is some pictures

<img width="1920" height="1080" alt="Screenshot (190)" src="https://github.com/user-attachments/assets/503ad752-8d24-4c42-aed6-cc100877f276" />


if you change Size to  400, 290 you can get 14 episodes in


<img width="1920" height="1080" alt="Screenshot (192)" src="https://github.com/user-attachments/assets/2c922254-747c-4dbe-9e55-bc8e23ba10da" />
